dspace/modules/additions/src/main/java/org/dspace/submit/step/ProjectStep.java

```diff
@@ -148,7 +142,11 @@ private int processProjectField(Context context, HttpServletRequest request, Ite
                 String funderAuthority = request.getParameter(MetadataField.formKey("rioxxterms", "funder", null) + "_authority");
                 try {
                     if (StringUtils.isBlank(funderAuthority)) {
-                        funderAuthority = new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).retrieveDefaultFunder(context).getId();
+                        FunderAuthorityValue defaultAuthority = new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).retrieveDefaultFunder(context);
+
+                        if(defaultAuthority!=null) {
+                            funderAuthority = defaultAuthority.getId();
+                        }
                     }
                     ProjectAuthorityValue project = projectService.createProject(context, value, funderAuthority);
```

Comment:
What will happen here if no default funder is configured? Do we create a project with a blank funder?